### ISSUE TITLE
Unable to register, login, or reset my password - Email dispatch failed

### ISSUE THREAD

**Issue by candideu** [souramoo/commentoplusplus#71]
Hello, thank you for keeping this project going!

I've deployed Commento++ using Heroku. The first time, I didn't add any SMTP settings, and app didn't deploy properly (there was no styling on the home page, and when I clickedd on "Create an account", the page was not found).

I deleted that app and tried again with SMTP credentials from Mailersend: 
![Screenshot_20220214-104939](https://user-images.githubusercontent.com/55474996/153897758-5ca34dd2-dde3-4188-be65-b9894eca6576.jpg)

However, when I created my account, I received the error `Email dispatch failed. pleasr contact support to resolve the issue.`

When I tried to login, I got this error:
![Screenshot_20220214-103318](https://user-images.githubusercontent.com/55474996/153895811-2592f1ce-4856-4e05-81b6-6c1568e30015.jpg)

When I tried to create an account with the same credentials,  I got this:
![Screenshot_20220214-103440](https://user-images.githubusercontent.com/55474996/153895809-44c89622-93bc-4eab-85f3-4c893bf5246c.jpg)

When I tried to reset my password, I got this error:
![Screenshot_20220214-103234](https://user-images.githubusercontent.com/55474996/153895812-10784b98-2b34-4ed5-a3f7-9d94552846b2.jpg)

**Comment by souramoo** [comment]
Thanks for the report! So what is happening here is that the email has been registered on the system but the account is unverified pending email confirmation. However the emails aren't sending so it isn't possible to verify the account! (hence the email is already registered message). The correct way out of this loop is to do a forgot password as you have done once the smtp issue has been fixed.

Do you have any log output please from commento about the email sending and the reason given for it failing? It appears there is some error in connecting to mailersend butwould be helpful to know what it is. Have you tried your mailersend details on any other program e.g. an email client to confirm they work to send emails? You can normally see this on heroku/other service you have deployed to.

**Comment by candideu** [comment]
I've solved it!

As you mentioned, I tested out my SMTP in a separate application (the newsletter app [Listmonk](https://listmonk.app/)). I checked the logs, and figured out that it was an issue with the `COMMENTO_SMTP_FROM_ADDRESS` config var.

When I set up Mailersend, I created a subdomain (ms.domain.com), but I set the SMTP from address in Comento++ using my apex domain (email@domain.com). Per Listmonk, Mailersend was flagging it as an unverified domain.

I fixed the config var in Commento++ so that it was email@ms.domain.com, and I was able to reset my password and login.

Thank you so much for your quick reply and help! I've been spreading the news about this fork so that it gets more support. This thread might be of interest: https://gitlab.com/commento/commento/-/issues/414#note_842239231
